src/Grid.php

```diff
@@ -87,6 +89,13 @@ public function init()
         }
     }
 
+    /**
+     * Add new column to grid. If column with this name already exists,
+     * an.
+     *
+     * @param string $name      New or existing column name
+     * @param [type] $columnDef [description]
```

Comment:
incorrect @param

Reply:
actually must be refactored to match Table

Reply:
done.